src/main/java/seedu/address/storage/JsonAdaptedNote.java

```diff
@@ -62,6 +66,20 @@ public boolean isClientNote() {
      * @throws IllegalValueException if there were any data constraints violated in the adapted note.
      */
     public Note toModelType() throws IllegalValueException {
+        if (contents == null) {
+            throw new IllegalValueException(
+                    String.format(MISSING_FIELD_MESSAGE_FORMAT, Note.class.getSimpleName()));
```

Comment:
I think this should be 'content' instead of Note.class.getSimpleName(), if not the message will read: Note's Note field is missing

Reply:
done